### PR TITLE
Location info on Cards and in full event views

### LINKED ISSUE #319: Design & rework venue address display
Currently, we don't display any event address information on the cards. We could consider displaying city and/or state somewhere prominent, and perhaps full address information somewhere less prominent.

We might also consider using map link data to display a map, sort of like @jswank does [here](https://observablehq.com/d/3a17b8a83df81b83#cell-346).

Places to consider location data redesign:
* Event card
* Event page
* Venue input
* Admin venue editing

### LINKED ISSUE #359: Fix text alignment on "More Info" and "Add to Calendar" Buttons in Mobile View
Vertical text alignment in mobile view is off by several pixels. 

Quick hack fix: set `class="card-btn add-to-calendar"` attributes to `padding-top:4px `
Need to investigate if there is a more graceful solution.

Screen Shot 2023-02-05 at 7 42 37 AM

### PR DESCRIPTION
**Pull request by Digi-D**
Show city location only from the specific event info on the cards on the main page. 
Show full address from the specific event on the event page

### CODE REVIEW

File: web-portal/components/EventView.vue [hunk 1]
```diff
@@ -188,6 +193,26 @@
...
+          // (but don't include state w/o city)
+          if (venue.city) {
+            addr += ' ' + venue.state ? `${venue.city}, ${venue.state}` : venue.city
+            // and only include zip if city, state is known
+            if (venue.zip) addr += ' ' + venue.zip
+          }
+        }
+
+        return addr
+      },
```
Comment (MatthewGidcomb) [inline web-portal/components/EventView.vue]: It looks like we forgot to actually wire this in on the template -- it's still building it up manually (lines 17-19).

File: web-portal/components/EventCard.vue [hunk 2]
```diff
@@ -141,6 +144,7 @@
           this.calendar_event.tags.includes('condition:sold-out')
       },
       isRemote: function () {
+        console.log(this.calendar_event.title + ' - ' + JSON.stringify(this.calendar_event.tags))
```
Comment (chriswininger) [inline web-portal/components/EventCard.vue]: we should delete this before we check in. It's not great to leave in console logs, they can degrade performance if nothing esle

**Review by chriswininger — APPROVED** [review]
I mostly yield to Mathew's approval for frontend but +1 from me with one minor suggestion

File: web-portal/components/EventView.vue (hunk 1 above)
Comment (MatthewGidcomb) [reply web-portal/components/EventView.vue]: That was my mistake -- fixed

File: web-portal/components/EventCard.vue (hunk 2 above)
Comment (MatthewGidcomb) [reply web-portal/components/EventCard.vue]: Removed, thanks for flagging.

**Review by MatthewGidcomb — APPROVED** [review]

**Comment by MatthewGidcomb** [comment]
Will close #319 and #359.

**Pull request merged**